src/Models/ShallowWaterModels/shallow_water_advection_operators.jl

```diff
@@ -98,9 +122,12 @@ the horizontal divergence of the velocity field U = (u, v), c ∇·(U),
 
 which will end up at the location `ccc`.
 """
-@inline function c_div_U(i, j, k, grid, solution, c)
-    @inbounds c[i, j, k] * 1/Vᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, u, solution) + δyᵃᶜᵃ(i, j, k, grid, v, solution))
-end
+@inline c_div_U(i, j, k, grid, solution, c, formulation) = 
+    @inbounds c[i, j, k] * 1/Azᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, u, solution) + δyᵃᶜᵃ(i, j, k, grid, v, solution))
```

Comment:
This also seems to have the wrong units, ie we need to multiply by the length in the perpendicular direction.

Reply:
> We can do that? That would be amazing to see, and so much faster than the hydrostatic model.

Haha! Why do you say that?